src/scripts/modules/media/body/content-baked.less

```diff
@@ -245,6 +245,14 @@ h1.example-title .text {
   }
 }
 
+[data-type="example"] {
+  .os-solution-container {
+    p {
+      margin-top: 2rem;
```

Comment:
This will add margin to all `p` inside `.os-solution-container`.
```suggestion
      .os-solution-container { margin-top: 2rem; }
```

Please consider that this issue is affecting not only paragraphs:
![screenshot_2018-12-20 intermediate algebra - openstax cnx](https://user-images.githubusercontent.com/31478212/50280519-0686c800-044d-11e9-950c-692bbcdde5da.png)

Reply:
I've added margin to `p` because there's also no gap between the solutions:
![image](https://user-images.githubusercontent.com/40228907/50281090-b01a8900-044e-11e9-9e37-505243af6255.png)
As far as I've checked, all solutions are inside a `p` tag or start with a `p` tag.  There are some that start with a table, but it looks fine.
So still we would need to add a margin to `p`, because of the few versions of solutions + eventually a margin to a whole `.os-solution.container`
`.os-solution-container {
    margin-top: 1rem;
    p {
      margin-top: 1rem;
    }
  }`
What do you think @PiotrKozlowski ?

Reply:
I think that will be good solution